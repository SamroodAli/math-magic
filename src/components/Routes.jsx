import { Segment } from 'semantic-ui-react';
import { Route, Switch } from 'react-router-dom';
import Calculator from './Calculator';
import Quotes from './Quotes';
import Home from './Home';

const Routes = () => (
  <Segment>
    <Switch>
      <Route path="/calculator" component={Calculator} />
      <Route path="/quotes" component={Quotes} />
      <Route path="/" component={Home} />
    </Switch>
  </Segment>
);

export default Routes;
