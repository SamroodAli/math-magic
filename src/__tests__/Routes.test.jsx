import renderer from 'react-test-renderer';
import Routes from '../components/Home';

test('Display panel displays all calculator buttons', () => {
  const tree = renderer.create(<Routes />).toJSON();
  expect(tree).toMatchSnapshot();
});
