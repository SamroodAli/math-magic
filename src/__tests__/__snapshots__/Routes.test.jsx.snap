// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes snapshot testing 1`] = `
<h1>
  Welcome to Math Magic
</h1>
`;
